src/commands/setup.rs

```diff
@@ -31,7 +31,12 @@ impl Setup {
 
         let network_options = match network::types::NetworkOptions::load(&input_file) {
             Ok(opts) => opts,
-            Err(e) => panic!("{}", e),
+            Err(e) => {
+                return Err(Box::new(network::error::NetavarkError {
+                    error: format!("Netavark Error {}", e),
+                    errno: -1,
```

Comment:
Should this be the exit code? If so it should be greater than 0.

Reply:
I think we should decide on some common error code it code is not already present. I suggested for `-1` if error code is not known. But i think we can move to something standard.